starter-code/src/clue.js

```diff
@@ -1,94 +1,220 @@
+const rope = {
+  name: "rope",
+  weight: 10
+};
 
-profPlum
-first_name:   Victor
-last_name:    Plum
-color:        purple
-description:  Billionaire video game designer
-age:          22
-image:        https://66.media.tumblr.com/ee7155882178f73b3781603f0908617c/tumblr_phhxc7EhPJ1w5fh03_540.jpg
-occupation:   Designer
+const knife = {
+  name: "knife",
+  weight: 8
+};
 
-missScarlet
-first_name:   Kasandra
-last_name:    Scarlet
-color:        red
-description:  She is an A-list movie star with a dark past
-age:          31
-image:        https://www.radiotimes.com/uploads/images/Original/111967.jpg
-occupation:   Actor
+const candlestick = {
+  name: "candlestick",
+  weight: 2
+};
 
-mrsPeacock
-first_name:   Eleanor
-last_name:    Peacock
-color:        blue
-description:  She is from a wealthy family and uses her status and money to earn popularity
-age:          36
-image:        https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg
-occupation:   Socialité
+const dumbbell = {
+  name: "dumbbell",
+  weight: 30
+};
 
-mrMustard
-first_name:   Jack
-last_name:    Mustard
-color:        yellow
-description:  He is a former football player who tries to get by on his former glory
-age:          62
-image:        https://static.independent.co.uk/s3fs-public/thumbnails/image/2016/07/04/08/unspecified-3.jpg
-occupation:   Retired Football player
+const poison = {
+  name: "dumbbell",
+  weight: 2
+};
 
-// Weapons
+const axe = {
+  name: "axe",
+  weight: 15
+};
 
-name: rope        ---   weight: 10
-name: knife       ---   weight: 8
-name: candlestick ---   weight: 2
-name: dumbbell    ---   weight: 30
-name: poison      ---   weight: 2
-name: axe         ---   weight: 15
-name: bat         ---   weight: 13
-name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
+const bat = {
+  name: "bat",
+  weight: 13
+};
 
-// Rooms
-name: Dining Room
-name: Conservatory
-name: Kitchen
-name: Study
-name: Library
-name: Billiard Room
-name: Lounge
-name: Ballroom
-name: Hall
-name: Spa
-name: Living Room
-name: Observatory
-name: Theater
-name: Guest House
-name: Patio
-*/
+const trophy = {
+  name: "trophy",
+  weight: 25
+};
+
+const pistol = {
+  name: "pistol",
+  weight: 20
+};
+
+const DiningRoom = {
+  name: "Dining Room"
+};
+const Conservatory = {
+  name: "Conservator"
+};
+const Kitchen = {
+  name: "Kitchen"
+};
+const Study = {
+  name: "Study"
+};
+const Library = {
+  name: "Library"
+};
+const BilliardRoom = {
+  name: "BilliardRoom"
+};
+const Lounge = {
+  name: "Lounge"
+};
+const Ballroom = {
+  name: "Ballroom"
+};
+const Hall = {
+  name: "Hall"
+};
+const Spa = {
+  name: "Spa"
+};
+const LivingRoom = {
+  name: "LivingRoom"
+};
+const Observatory = {
+  name: "Observatory"
+};
+const Theater = {
+  name: "Theater"
+};
+const GuestHouse = {
+  name: "GuestHouse"
+};
+const Patio = {
+  name: "Patio"
+};
 
 // Characters Collection
 const charactersArray = [];
-
+charactersArray.push(
```

Comment:
You could have just added these elements directly int he array, instead of pushing them into the array.